packages/components/src/border-control/border-control/component.tsx

```diff
@@ -67,6 +68,13 @@ const UnconnectedBorderControl = (
 		...otherProps
 	} = useBorderControl( props );
 
+	if ( showDropdownHeader !== undefined ) {
+		deprecated( 'wp.components.BorderControl `showDropdownHeader` prop', {
+			since: '6.7',
+			version: '7',
```

Comment:
If we want to specify a "to be removed in" version, let's make it follow the same pattern as everywhere, since each "semver major + minor" WordPress version is considered major.

```suggestion
			version: '7.0',
```

Reply:
I don't quite understand this warning though. We already removed the functionality in this branch so the removal is effective immediately, not in 7.0.

Personally, I prefer not logging a warning at all, because there's really no action that a consumer could take, other than stop passing the prop. And there's no reason to stop passing the prop, hence our decision to keep the TS compiling for convenience.

If you're worried about one day wanting to remove the TS definition, I think we're free to remove that at any time without warning.

Reply:
Makes sense, I removed the warning before merge 👍